### PR TITLE
Shibboleth extensions

### PR DESCRIPTION
**Pull request by dajuno**
# MaRDI Pull Request

**Instructions for PR review**:
- [ ] Conceptual Review (Logic etc...) 
- [ ] Code Review (Review your implementation) 
- [ ] Checkout (Test changes locally) 

**Checklist for this PR**: 
- [ ] [Reviewers and Assignee specified.](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)
- [ ] [All related issues are linked.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

### CODE REVIEW

File: mediawiki/LocalSettings.d/PluggableAuth.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+# https://www.mediawiki.org/wiki/Extension:PluggableAuth
+wfLoadExtension( 'PluggableAuth' );
+
+$wgPluggableAuth_Config = array(
+    array(
```
Comment (physikerwelt) [inline mediawiki/LocalSettings.d/PluggableAuth.php:6]: in php 7 you can just write [ instead of array

**Review by physikerwelt — APPROVED** [review]

**Comment by physikerwelt** [comment]
I am a bit skeptical about this one, as the tests repeatedly fail, whereas other pull requests went through. @eloiferrer maybe you can cross-check locally before we destroy anything with this.

**Comment by dajuno** [comment]
> I am a bit skeptical about this one, as the tests repeatedly fail, whereas other pull requests went through. @eloiferrer maybe you can cross-check locally before we destroy anything with this.

Not eloi but I just ran the checks locally without issue (after completely wiping my docker environment).

**Comment by physikerwelt** [comment]
Okay, then let's merge it.

**Pull request merged**